db/migrate/20151101142959_collaborations.rb

```diff
@@ -0,0 +1,12 @@
+class Collaborations < ActiveRecord::Migration
+  def change
+    create_table :collaborations do |t|
+      t.integer :user_id
+      t.integer :wiki_id
+      t.timestamps
+    end
+    add_index :collaborations, :id, unique: true
+    add_index :collaborations, :user_id
+    add_index :collaborations, :wiki_id
```

Comment:
Consider replacing the `:user_id` and `:wiki_id` indexes with a multi-column unique index in a new migration.

The multi-column unique index will prevent a user from being added as a collaborator to the same wiki more than once.

```ruby
  remove_index :collaborations, :user_id
  remove_index :collaborations, :wiki_id
  add_index :collaborations, [:user_id, :wiki_id], unique: true
```